newIDE/app/src/Profile/Achievement/UserAchievements.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import { Trans } from '@lingui/macro';
+import React, { useState, useEffect, useCallback } from 'react';
+
+import { Column } from '../../UI/Grid';
+import { ResponsiveLineStackLayout } from '../../UI/Layout';
+import Text from '../../UI/Text';
+import {
+  getAchievements,
+  type Badge,
+  type Achievement,
+} from '../../Utils/GDevelopServices/User';
+
+import AchievementList from './AchievementList';
+// $FlowFixMe[missing-export] TODO: check that this is the correct way to import SVG in the project. If yes, follow https://github.com/facebook/flow/issues/7121.
```

Comment:
We don't import SVG directly but use Material-UI "SvgIcon" component that wrap SVG for us. 
For example, look at newIDE/app/src/UI/CustomSvgIcons/Chrome.js

See usage in the doc:
https://v4.mui.com/components/icons/#svgicon
https://v4.mui.com/api/svg-icon/

Reply:
Really clear, thanks!